lib/SPIRV/libSPIRV/SPIRVEntry.cpp

```diff
@@ -38,6 +38,7 @@
 //===----------------------------------------------------------------------===//
 
 #include "SPIRVEntry.h"
+#include "SPIRVAsm.h"
```

Comment:
```suggestion
```

It is unclear why this header is needed here: file wasn't changed at all

Reply:
I had to rollback this because build fails. Looks like we really need this header

Reply:
That is because this file contains
`#include "SPIRVOpCodeEnum.h"`
Which itself cannot contain any headers, but in the same time requires SPIRVAsm.h to work